### ISSUE TITLE
Many different small UI-related changes

### ISSUE THREAD

**Issue by mawoka-myblock** [mawoka-myblock/ClassQuiz#118]
## Play-Screen
- [x] Ability to change background-color (@mawoka-myblock)
### Admin
- [x] Move buttons from bottom into top-row, like navbar (@mawoka-myblock)
- [ ] Play audio-file on "Show next question"-button
- [ ] #210
- [x] Better Result-Screens (@mawoka-myblock)
- [x] Improve Timer (@mawoka-myblock)

### Player
- [x] Improve Timer (@mawoka-myblock)
- [x] Redesign buttons in kahoot-mode (@mawoka-myblock)

**Comment by artimanputra** [comment]
I would like to work on this issue as a part of hacktoberfest, Can I work on this?

**Comment by mawoka-myblock** [comment]
> I would like to work on this issue as a part of hacktoberfest, Can I work on this?

Yes! Of course! All of them? If not, which? If you have and difficulties, just contact me!

**Comment by mawoka-myblock** [comment]
@artimanputra Sorry that I've already worked on these things, but I wanted to finish them and you didn't reply in time (for me). Sorry for that, but you'll find some rough edges in ClassQuiz, for sure, so you can still go ahead and fix them.

**Comment by mawoka-myblock** [comment]
@artimanputra Anything new? Otherwise, I'll work on these issues.

**Comment by aliaagheisX** [comment]
@mawoka-myblock  I can work on the rest of the tasks

**Comment by mawoka-myblock** [comment]
> @mawoka-myblock I can work on the rest of the tasks

Then, go ahead!

**Comment by aliaagheisX** [comment]
@mawoka-myblock  i think `Show question and show answers only after some time, like Kahoot!` added already

**Comment by mawoka-myblock** [comment]
> @mawoka-myblock i think `Show question and show answers only after some time, like Kahoot!` added already

Where?

**Comment by aliaagheisX** [comment]
> * [ ]  Show question and show answers only after some time, like Kahoot!
> *

sorry I think I had misunderstood the task at the first place, I  won't be able to work on that task. Sorry for that

**Comment by mawoka-myblock** [comment]
> > * [ ]   Show question and show answers only after some time, like Kahoot!
> 
> sorry I think I had misunderstood the task at the first place, I won't be able to work on that task. Sorry for that

No problem!

**Comment by trygve55** [comment]
Hello, is it possible to add "Automatically go to results when everyone have answered a question". Thank you.

**Comment by mawoka-myblock** [comment]
Yes, @trygve55! I like the idea, but please open a new issue for that and I'll look into that, because it isn't as easy as it seems.

**Comment by mawoka-myblock** [comment]
Any progress, @aliaagheisX?
